src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -57,10 +93,20 @@ export default {
 	},
 	data() {
 		return {
+			email: null,
+			isGuestCheckout: false,
```

Comment:
I think this needs to be a prop...don't see anything internal to this component that would alter this value.